library/general/test/popup_test.rb

```diff
@@ -14,26 +14,46 @@
   end
 
   describe ".Feedback" do
-    before do
-      expect(ui).to receive(:OpenDialog)
-      expect(ui).to receive(:CloseDialog)
-      allow(ui).to receive(:BusyCursor)
-      allow(ui).to receive(:GetDisplayInfo).and_return({})
+    context "when arguments are good" do
```

Comment:
Most of this is whitespace. See <https://github.com/yast/yast-yast2/pull/572/files?w=1>  for a better view.